### PR TITLE
bugfix/CD-121: Styling login/register

### PR DESCRIPTION
**Pull request by gusartus**

### CODE REVIEW

**Comment by CodeAnnihilator** [comment]
image

File: src/locales/en.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
 		"OrSignUpWith": "Or sign up with",
 		"BecomeMember": "Become a member",
 		"AlreadyHaveAccount": "Already have an account",
-		"Registation": {
+		"registration": {
```
Comment (Gerasimov94) [inline src/locales/en.json]: you broke a realisation in this place

File: src/modules/RegAuth/Auth/components/auth.scss [hunk 2]
```diff
@@ -63,16 +77,28 @@
...
 }
 
+.orUseEmail {
+	position: relative;
+	transform: translateY(-50%);
+	padding: 10px 40px;
+	color: #56585D;
+	font-size: 12px;
+	top: -1px;
+	background-color: #FFFFFF;
```
Comment (Gerasimov94) [inline src/modules/RegAuth/Auth/components/auth.scss]: is $white

File: src/modules/RegAuth/Registration/components/registration.scss [hunk 3]
```diff
@@ -45,6 +53,16 @@
 	align-items: center;
 }
 
+.orSignUpWith {
+	position: relative;
+	transform: translateY(-50%);
+	padding: 10px 30px;
+	color: #56585D;
```
Comment (Gerasimov94) [inline src/modules/RegAuth/Registration/components/registration.scss]: multiply usind of this color. move into a .colors

**Review by Gerasimov94 — CHANGES_REQUESTED** [review]

**Comment by CodeAnnihilator** [comment]
image
label component has never been merged with master yet

**Pull request merged**